docs/core/telemetry.md

```diff
@@ -7,16 +7,17 @@ order: 10
 Gather relevant insights about your application and modules with custom metrics and telemetry. {synopsis}
 
 The Cosmos SDK enables operators and developers to gain insight into the performance and behavior of
-their application through the use of the `telemetry` package. The Cosmos SDK currently supports
-enabling in-memory and prometheus as telemetry sinks. This allows the ability to query for and scrape
-metrics from a single exposed API endpoint -- `/metrics?format={text|prometheus}`, the default being
-`text`.
+their application through the use of the `telemetry` package. To enable telemetrics, set `telemetry.enabled = true` in the app.toml config file.
+
+The Cosmos SDK currently supports enabling in-memory and prometheus as telemetry sinks. In-memory sink is always attached (when the telemetry is enabled) with 10 second interval and 1 minute retention. This means that metrics will be aggregated over 10 seconds, and metrics will be kept alive for 1 minute.
```

Comment:
IIRC, the retention is configurable BTW.

Reply:
It's configurable for Prometheus, but not for memory sink.

Reply:
Naturally, so we shouldn't state these as static. Maybe state that by default they are these values.

Reply:
It's hardcoded - so it's static. https://github.com/cosmos/cosmos-sdk/blob/robert%2Fquery-telemetry/telemetry/metrics.go#L92